src/components/30-organisms/commercial-hero-banner/child.scss

```diff
@@ -13,13 +13,27 @@ axa-commercial-hero-banner {
     overflow-wrap: break-word;
   }
 
-  p {
+  p, .checkmarks {
     @include text(t4);
     @include breakpoint($mediaquery-xs-up) {
       @include text(t3);
     }
   }
 
+  .checkmark {
+    margin-bottom: 5px;
```

Comment:
If you would make `.checkmark` with `display: flex; justify-content: center;` ....